### DISCUSSION TITLE
Two questions

### DISCUSSION THREAD

**Discussion by EPFAN** [valinet/ExplorerPatcher#1780] [Q&A]
When I install EP the on-screen-volume for system sound will not appear with my keyboard volume button. Secondly, I like to keep the Registry Edit tool in my Start Menu but when I click it and try to export a Key, the save-window disappears instantly.

Need some guidance. Thank you.

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Reply by EPFAN** [reply]
Windows 11 Home
Version: 22H2
Build: 22621.2070

EP: 22621.1992.56.1

**Comment by pyrates999** [comment]
That is a preview update: https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

**Comment by vippium** [comment]
Currently, this is a bug with insider builds. No fix yet

**Reply by pyrates999** [reply]
They're not running an insider build, just have a preview update installed.

**Reply by vippium** [reply]
Yup, but they both are very similar... Means you want new features before official stable release. That's why, i've mentioned them as insider.

**Comment by EPFAN** [comment]
Can someone point me to the older versions. Seems two version back I didn't have this problem.  TIA

**Reply by pyrates999** [reply]
This version: https://support.microsoft.com/en-au/topic/july-11-2023-kb5028185-os-build-22621-1992-605fa18f-bd49-41d8-80b1-245080e26c3d

**Reply by EPFAN** [reply]
I mean two version back of EP. Is there an (older version) directory here at github.com?

**Reply by pyrates999** [reply]
All releases are here: https://github.com/valinet/ExplorerPatcher/releases

**Reply by EPFAN** [reply]
Thanks for your help. I tried several older versions and they work great, just like the newest version. However, something to do with a Windows Update (my thoughts) is not allowing W11 to work in harmony with EP. Meaning:

Both the volume display when using the keyboard control volume control button does not display and exporting registry keys is not possible because the save registry key windows does not open.

I have also been using Start11 from stardock.com since W10. When I upgraded to W11 on this machine Start10 needed to be removed and replaced with Start11. Start 11 has no negative effects on W11 features. Sadly, on W11, when EP is installed the two glitches I mentioned become prevalent.

Down the road when I do a clean install of W11 or upgrade to the next Windows OS to be released in 2024, EP not block either of the two features I have talked about.

Thanks for your help, pyrates999. I will be leaving this discussion now and will return in a few months to get the newest release of EP to see how it works.

**Reply by pyrates999** [reply]
You got a preview update installed: https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.
